src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Figure implements Area, Output{
+    double a, b;
+
+    @Override
+    public double findArea() {
+        return a * b;
+    }
+
+    @Override
+    public String printAllInformation() {
+        return null;
```

Comment:
Don't return 'null' from a method. Instead of returning 'null', provide a meaningful default value or throw an exception if applicable.